2024-04-02 17:05:19,715 INFO    MainThread:16112 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-04-02 17:05:19,715 INFO    MainThread:16112 [wandb_setup.py:_flush():76] Configure stats pid to 16112
2024-04-02 17:05:19,715 INFO    MainThread:16112 [wandb_setup.py:_flush():76] Loading settings from C:\Users\¿Ó≥–≥œ\.config\wandb\settings
2024-04-02 17:05:19,716 INFO    MainThread:16112 [wandb_setup.py:_flush():76] Loading settings from E:\lesson\MLP\MLP_Test Task\src\wandb\settings
2024-04-02 17:05:19,716 INFO    MainThread:16112 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-02 17:05:19,716 INFO    MainThread:16112 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-02 17:05:19,716 INFO    MainThread:16112 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'src\\main.py', 'program_abspath': 'E:\\lesson\\MLP\\MLP_Test Task\\src\\main.py', 'program': 'E:\\lesson\\MLP\\MLP_Test Task\\src\\main.py'}
2024-04-02 17:05:19,717 INFO    MainThread:16112 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-02 17:05:19,717 INFO    MainThread:16112 [wandb_init.py:_log_setup():527] Logging user logs to E:\lesson\MLP\MLP_Test Task\src\wandb\run-20240402_170519-w132z6rv\logs\debug.log
2024-04-02 17:05:19,717 INFO    MainThread:16112 [wandb_init.py:_log_setup():528] Logging internal logs to E:\lesson\MLP\MLP_Test Task\src\wandb\run-20240402_170519-w132z6rv\logs\debug-internal.log
2024-04-02 17:05:19,717 INFO    MainThread:16112 [wandb_init.py:init():567] calling init triggers
2024-04-02 17:05:19,718 INFO    MainThread:16112 [wandb_init.py:init():574] wandb.init called with sweep_config: {}
config: {}
2024-04-02 17:05:19,718 INFO    MainThread:16112 [wandb_init.py:init():592] re-initializing run, found existing run on stack: be8kwvvi
2024-04-02 17:05:19,718 INFO    MainThread:16112 [wandb_run.py:_finish():2052] finishing run duisburg-essen-university/MLP/be8kwvvi
2024-04-02 17:05:19,718 INFO    MainThread:16112 [wandb_run.py:_atexit_cleanup():2292] got exitcode: 0
2024-04-02 17:05:19,719 INFO    MainThread:16112 [wandb_run.py:_restore():2275] restore
2024-04-02 17:05:19,719 INFO    MainThread:16112 [wandb_run.py:_restore():2281] restore done
2024-04-02 17:05:28,203 INFO    MainThread:16112 [wandb_run.py:_footer_history_summary_info():3920] rendering history
2024-04-02 17:05:28,204 INFO    MainThread:16112 [wandb_run.py:_footer_history_summary_info():3952] rendering summary
2024-04-02 17:05:28,226 INFO    MainThread:16112 [wandb_run.py:_footer_sync_info():3879] logging synced files
2024-04-02 17:05:28,244 INFO    MainThread:16112 [wandb_init.py:init():617] starting backend
2024-04-02 17:05:28,244 INFO    MainThread:16112 [wandb_init.py:init():621] setting up manager
2024-04-02 17:05:28,249 INFO    MainThread:16112 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-04-02 17:05:28,250 INFO    MainThread:16112 [wandb_init.py:init():629] backend started and connected
2024-04-02 17:05:28,259 INFO    MainThread:16112 [wandb_init.py:init():721] updated telemetry
2024-04-02 17:05:28,373 INFO    MainThread:16112 [wandb_init.py:init():754] communicating run to backend with 90.0 second timeout
2024-04-02 17:05:30,401 INFO    MainThread:16112 [wandb_run.py:_on_init():2344] communicating current version
2024-04-02 17:05:31,370 INFO    MainThread:16112 [wandb_run.py:_on_init():2353] got version response 
2024-04-02 17:05:31,370 INFO    MainThread:16112 [wandb_init.py:init():805] starting run threads in backend
2024-04-02 17:05:39,207 WARNING MainThread:16112 [wandb_init.py:init():1209] interrupted
Traceback (most recent call last):
  File "E:\anaconda3\envs\MLP\lib\site-packages\wandb\sdk\wandb_init.py", line 1187, in init
    run = wi.init()
  File "E:\anaconda3\envs\MLP\lib\site-packages\wandb\sdk\wandb_init.py", line 818, in init
    run_start_result = run_start_handle.wait(timeout=30)
  File "E:\anaconda3\envs\MLP\lib\site-packages\wandb\sdk\lib\mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "E:\anaconda3\envs\MLP\lib\site-packages\wandb\sdk\lib\mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "E:\anaconda3\envs\MLP\lib\site-packages\wandb\sdk\lib\mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "E:\anaconda3\envs\MLP\lib\threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "E:\anaconda3\envs\MLP\lib\threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
